packages/expo-blur/src/BlurView.types.ts

```diff
@@ -15,6 +15,17 @@ export type BlurViewProps = {
    * @default 50
    */
   intensity?: number;
+  /**
+   * A number by which the blur intensity will be divided on Android.
+   *
+   * Due to platform differences blurs on Android and iOS vary slightly and might look different
+   * at different intensity levels. This property can be used to fine tune blur intensity on Android to match it
+   * more closely with iOS.
+   * @default 4
```

Comment:
```suggestion
   * @default 3.5
```
It's set to `3.5` in the native code, so I suspect that is the default value.

Reply:
Oops! I forgot to change the value in the native code to 4 (javascript was passing 4 anyways so I didn't notice)